contributors/devel/sig-node/e2e-node-tests.md

```diff
@@ -202,6 +196,46 @@ For example, the [`ci-kubernetes-node-kubelet`](https://github.com/kubernetes/te
 make test-e2e-node REMOTE=true FOCUS="\[NodeConformance\]" SKIP="\[Flaky\]|\[Serial\]"
 ```
 
+See [The Spec Runner](http://onsi.github.io/ginkgo/#the-spec-runner) in the Grinkgo documentation to learn more about how FOCUS and SKIP work.
+
+## Run a subset of tests using image configuration files
+
+Image configuration files can further influence FOCUS and SKIP match test cases.
+
+For example:
+
+```sh
+--focus="\[NodeFeature:.+\]" --skip="\[Flaky\]|\[Serial\]"
+```
+
+runs all e2e tests labeled `"[NodeFeature:*]"` and will skip any tests labeled `"[Flaky]"` or `"[Serial]"`.
+
+Two example e2e tests that would match this expression are:
+  * https://github.com/kubernetes/kubernetes/blob/0e2220b4462130ae8a22ed657e8979f7844e22c1/test/e2e_node/security_context_test.go#L155
+  * https://github.com/kubernetes/kubernetes/blob/0e2220b4462130ae8a22ed657e8979f7844e22c1/test/e2e_node/security_context_test.go#L175
+
+However, image configuration files select test cases based on the `tests` field. See [this](https://github.com/kubernetes/test-infra/blob/4572dc3bf92e70f572e55e7ac1be643bdf6b2566/jobs/e2e_node/benchmark-config.yaml#L22-23) for an example configuration.
+
+If the [Prow e2e job configuration](https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/image-config.yaml) does **not** specify the `tests` field, FOCUS and SKIP will run as expected.
+
+## Run a single test
+
+To run a particular e2e test, simply pass the Grinkgo `It` string to the `--focus` argument.
+
+For example, suppose we have the following [test case](https://github.com/kubernetes/kubernetes/blob/0e2220b4462130ae8a22ed657e8979f7844e22c1/test/e2e_node/security_context_test.go#L175). We could select this test case by adding the argument:
+
+```sh
+--focus="should not show its pid in the non-hostpid containers \[NodeFeature:HostAccess\]" --skip="\[Flaky\]|\[Serial\]"
+```
+
+## Run all tests related to a feature
+
+In contrast, to run all node e2e tests related to the "HostAccess" feature one could run:
+
+```sh
+--focus="\[NodeFeature:HostAccess\]" --skip="\[Flaky\]|\[Serial\]"
```

Comment:
Same here. We should either remove the skip flag here or mention it in the explanation.

Reply:
Removed skip flag